scripts/tests/test_zkai_native_attention_mlp_adapter_air_frontier_gate.py

```
@@ -0,0 +1,162 @@
+        )
+
+    def test_mutations_reject_overclaims_and_drift(self) -> None:
+        payload = self.fresh_payload()
+        self.assertTrue(payload["all_mutations_rejected"])
+        self.assertEqual(payload["case_count"], len(gate.MUTATION_NAMES))
+        self.assertEqual([case["name"] for case in payload["cases"]], list(gate.MUTATION_NAMES))
+
+        mutated = self.fresh_payload()
+        mutated["summary"]["native_adapter_air_proven_in_current_artifact"] = True
+        gate.refresh_payload_commitment(mutated)
+        with self.assertRaises(gate.AdapterAirFrontierError):
```

Ruff (0.15.12)

[warning] Use `pytest.raises` instead of unittest-style `assertRaises`

Replace `assertRaises` with `pytest.raises`

(PT027)